aesara/link/numba/dispatch/elemwise.py

```diff
@@ -164,6 +164,18 @@ def create_vectorize_func(
     return elemwise_fn
 
 
+def normalize_axis(axis, ndim):
+    if axis is None:
+        return axis
+
+    if axis < 0:
+        axis = ndim + axis
+
+    if axis < 0 or axis >= ndim:
+        raise np.AxisError(ndim=ndim, axis=axis)
+    return axis
```

Comment:
Can you just use https://github.com/numpy/numpy/blob/7f0f045625022c3f816911cd80f8635ac2a36f21/numpy/core/numeric.py#L1348

Reply:
Dod you decide not to use these?

Reply:
sorry, done